nixos/modules/security/pam.nix

```diff
@@ -848,9 +857,7 @@ let
             order = "user,group,default";
             debug = true;
           }; }
-          { name = "kwallet5"; enable = cfg.enableKwallet; control = "optional"; modulePath = "${pkgs.plasma5Packages.kwallet-pam}/lib/security/pam_kwallet5.so"; settings = {
-            kwalletd = "${pkgs.plasma5Packages.kwallet.bin}/bin/kwalletd5";
-          }; }
+          { name = "kwallet"; enable = cfg.kwallet.enable; control = "optional"; modulePath = "${cfg.kwallet.package}/lib/security/pam_kwallet5.so"; }
```

Comment:
Does `pam_kwallet5.so` name still holds for `kdePackages`/kwalelt6?

Reply:
_Yes_

Reply:
May I also suggest using the opportunity of adding the `security.pam.services.*.kwallet` sub-attrset, to add an option (or do it unconditionally) to include `auto_start force_run` settings to `session` , needed for kwallet integration to work with startx/startplasma-wayland-from-terminal/sddm-less launch?

Reply:
This can be done later, in a separate PR.